### ISSUE TITLE
[Tracking] Optimizing readability/performance for the insardia command.

### ISSUE THREAD

**Issue by insyri** [insyri/bhop-bot-ts#8]
## Issue Information
This issue will be held for optimization notes for the insardia command.

Goal: Less "wtfs" on the code; code clarity, readability, and removing unnecessary parts.

## Tracking
- [x] #9
- [x] #10
- [x] #11
- [x] #12

# P1: `alternatives` map
https://github.com/insyri/bhop-bot-ts/blob/50e59f91bd42771a9e6295c116944ea502042304/src/commands/fun/insardia.ts#L9-L15
I forgot why exactly I used a map here, I think this can be shortened into an object for less memory movement. Implementation would be rather easy as it would basically transform the Map to an Object using [`Object.fromEntries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries), [demo](https://tsplay.dev/w6X8Ym):
```ts
let alternatives = new Map();
alternatives.set("hi", ["hi", "hizies"]),
  alternatives.set("hey", ["hey", "heya"]),
  alternatives.set("and", ["and", "n"]),
  alternatives.set("you", ["yu", "you", "u"]),
  alternatives.set("are", ["are", "r"]),
  alternatives.set("got", ["got", "gto", "tog"]);

let alternativesobj = Object.fromEntries(alternatives);
console.log(alternativesobj)
```
And then overwrite the variable via coding not executing.

# P2: Lines
https://github.com/insyri/bhop-bot-ts/blob/50e59f91bd42771a9e6295c116944ea502042304/src/commands/fun/insardia.ts#L28-L30 
wtf.

@drumman22 If you remember, could you explain why you chose to split the entire string by the newlines or at least remember if this was a relevant string of code to implement? AFAIK this wouldn't change anything in output.

# P3: Recursive ifs
https://github.com/insyri/bhop-bot-ts/blob/50e59f91bd42771a9e6295c116944ea502042304/src/commands/fun/insardia.ts#L48-L60

All these checks lead up to a `continue`, I'm not sure what would change if it is removed, I would need to do some more research into what exactly continue does.

MDN reference for [`continue`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue).

# P4: Merging duplicate conditions
https://github.com/insyri/bhop-bot-ts/blob/50e59f91bd42771a9e6295c116944ea502042304/src/commands/fun/insardia.ts#L62-L74
Literally just:
```ts
if (index === 0 || index === word.length - 1) {
  if (getRng(50) === 1) new_word += letter + letter; // Some way to shorten letter + letter?
}
```

# P5: Merge P3 and P4
In short, put P4 inside of P3's condition checking, fundamentally replacing where `continue` is with P4.

# Cleanup
As of now, these are all the specific/critical issues to be resolved. Any more input or follow up is welcome to be commented.
